src/bite.v1.js

```diff
@@ -21,9 +22,41 @@
         request.send()
     }
 
+    function determineSession() {
+        const cookies = document.cookie
+            .split('; ')
+            .map(cookie => cookie.split('='))
+            .reduce((mapping, [key, value]) => ({
+                ...mapping,
+                [key]: value,
+            }), {})
+
+        if (cookies['barnebys_session']) {
```

Comment:
We might want to add a TTL for any existing sessions as well, not sure what the desired behaviour is

Reply:
TTL should be a build env so we easily can configure that

Reply:
Sure thing, I'll add it